backend/src/main/java/com/htecgroup/skynest/SkyNestApplication.java

```diff
@@ -40,4 +42,12 @@ public DriverConfigLoaderBuilderCustomizer defaultProfile() {
             .withString(DefaultDriverOption.METADATA_SCHEMA_REQUEST_TIMEOUT, "20 seconds")
             .build();
   }
+
+  @Bean("dropboxClient")
+  public DbxClientV2 dropboxClient() {
+    String ACCESS_TOKEN =
+        "sl.BLvQJNuOz5yfPt5TSENtKsPX53OUioAOpcZez0-cvC_zjDOv-wxnXhK6s6eQ__9J3FVUuAnkDzjVaZDQo6nz9Jrr991MWDmChdWLIn8gdZkPOhwgkjnlZAaGz9tKUpG0Ea7DrFY";
```

Comment:
> The ACCESS_TOKEN is currently defined directly in the creation of the bean. I had an issue where the token expired after a while. If the token doesn't expire and everything is okay, then it can go with the other .env variables and keep it there.

won't it expire anyway? what difference would it make if it were an env var?

Reply:
the only difference it makes is that I lose time if I put it in the env and it expires. Anyhow this is all changed now. It's not gonna be this way